docs/tctl.md

```diff
@@ -326,6 +326,6 @@ As you add the bad binary checksum to your namespace, Temporal will not dispatch
 
 `--tls_cert_path=<certificate file path>` command line argument that passes a certificate for the server to validate the client (`tctl`) identity. Requires that `--tls_key_path` is also provided.
 
-`--tls_key_path=<private key file path>` command line argument that passes a private key for secure communication with the server. Requires that `--tls_key_path` is also provided.
+`--tls_key_path=<private key file path>` command line argument that passes a private key for secure communication with the server.
```

Comment:
"command line argument that passes a private key for secure communication with the server." --> this still seems off / hard to grok. IMO we should say that this is the certificate private key associated with the client certificate supplied with the --tls_cert_path setting.